python/GafferUITest/TestCase.py

```diff
@@ -83,6 +83,21 @@ def assertExampleFilesExist( self ) :
 			expanded = os.path.expandvars( e['filePath'] )
 			self.assertTrue( os.path.exists( expanded ), "%s does not exist" % expanded )
 
+	def assertExampleFileDontReferenceUnstablePaths( self ) :
+
+		forbidden = (
+			"${scriptName}",
```

Comment:
Shouldn't this be `${script:name}`?

Reply:
_**Doh!**_ . sorry, some part of my brain must still be in the world of `<some other similar tool>`. Fixed in https://github.com/GafferHQ/gaffer/pull/3309/commits/ae141bf1daad86961ce23c67970679d35e78d5ed